site/docs/install.md

```diff
@@ -5,14 +5,17 @@ title: Installing Bazel
 
 # Installing Bazel
 
-See the instructions for installing Bazel on:
+The Bazel team officially supports Bazel on the following platforms. See the instructions for installing Bazel on:
```

Comment:
Maybe "the following" -> "these" to shorten?

Reply:
And maybe "see the instructions for installing Bazel on" could go entirely.